QtlDataPass/ContentView.swift

```diff
@@ -8,12 +8,29 @@
 import SwiftUI
 
 struct ContentView: View {
+    @State var textSecond = ""
+    @State var textThird = ""
```

Comment:
```suggestion
    @State private var textSecond = ""
    @State private var textThird = ""
    
```

[must] 他の画面からアクセスしない変数のはずなので```private```を付与してください

[nits] bodyとProperty Wrapperの間には改行を入れるとより見やすくなります

Reply:
コミットハッシュ： c2e095ab8f744f2d57332797fb7c2515a148b25e

変更： TopViewのプロパティ名とprivate付与　(13行目&16行目)